### DISCUSSION TITLE
Select multiple zones for alerts query.

### DISCUSSION THREAD

**Discussion by Orcazephyr** [weather-gov/api#623] [General]
Because there appears to be no easy way to get alerts for a city and its neighbouring marine zones, I have made a list of offices and their corresponding zones. However, there is a huge performance hit to query each zone separately through "https://api.weather.gov/alerts/active/zone/$zone." 
Is there a way to provide multiple zones to the API?

**Comment by dwhitemv25** [comment]
`/alerts/active` can take a list of zones to search if you use the parameter format:

```
https://api.weather.gov/alerts/active?zone=ORZ005,ORZ008&limit=500
```
